2012/08/19 02:07:08 [error] [php] date() [<a href='function.date'>function.date</a>]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'UTC' for '8.0/no DST' instead (D:\yii\webroot\protected\views\layouts\main.php:51)
Stack trace:
#0 D:\yii\framework\web\CBaseController.php(96): SiteController->renderInternal()
#1 D:\yii\framework\web\widgets\CContentDecorator.php(77): SiteController->renderFile()
#2 D:\yii\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate()
#3 D:\yii\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput()
#4 D:\yii\framework\web\CBaseController.php(207): CContentDecorator->run()
#5 D:\yii\framework\web\CBaseController.php(301): SiteController->endWidget()
#6 D:\yii\webroot\protected\views\layouts\column1.php(6): SiteController->endContent()
#7 D:\yii\framework\web\CBaseController.php(127): require()
#8 D:\yii\framework\web\CBaseController.php(96): SiteController->renderInternal()
#9 D:\yii\framework\web\CController.php(785): SiteController->renderFile()
#10 D:\yii\webroot\protected\controllers\SiteController.php(32): SiteController->render()
#11 D:\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#12 D:\yii\framework\web\CController.php(309): CInlineAction->runWithParams()
#13 D:\yii\framework\web\CController.php(287): SiteController->runAction()
#14 D:\yii\framework\web\CController.php(266): SiteController->runActionWithFilters()
#15 D:\yii\framework\web\CWebApplication.php(283): SiteController->run()
#16 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController()
#17 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#18 D:\yii\webroot\index.php(13): CWebApplication->run()
REQUEST_URI=/
in D:\yii\webroot\protected\views\layouts\main.php (51)
in D:\yii\webroot\protected\views\layouts\column1.php (6)
in D:\yii\webroot\protected\controllers\SiteController.php (32)
2012/09/02 09:49:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "gii".' in D:\yii\framework\web\CController.php:484
Stack trace:
#0 D:\yii\framework\web\CController.php(271): CController->missingAction('gii')
#1 D:\yii\framework\web\CWebApplication.php(283): CController->run('gii')
#2 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('site/gii')
#3 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 D:\yii\webroot\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/webroot/index.php?r=site/gii
---
2012/09/17 21:38:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "user".' in D:\yii\framework\web\CController.php:484
Stack trace:
#0 D:\yii\framework\web\CController.php(271): CController->missingAction('user')
#1 D:\yii\framework\web\CWebApplication.php(283): CController->run('user')
#2 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('site/user')
#3 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 D:\yii\webroot\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/webroot/index.php?r=site/user
---
2012/09/20 23:15:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "user".' in D:\yii\framework\web\CController.php:484
Stack trace:
#0 D:\yii\framework\web\CController.php(271): CController->missingAction('user')
#1 D:\yii\framework\web\CWebApplication.php(283): CController->run('user')
#2 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('site/user')
#3 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 D:\yii\webroot\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/webroot/index.php?r=site/user
---
2012/12/23 23:15:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "usr".' in D:\yii\framework\web\CWebApplication.php:287
Stack trace:
#0 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('usr')
#1 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\yii\webroot\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=usr
---
2012/12/23 23:16:41 [error] [exception.CException] exception 'CException' with message 'Property "User.email" is not defined.' in D:\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('email')
#1 D:\yii\framework\web\helpers\CHtml.php(2158): CActiveRecord->__get('email')
#2 D:\yii\framework\web\helpers\CHtml.php(1887): CHtml::resolveValue(Object(User), 'email')
#3 D:\yii\framework\web\helpers\CHtml.php(1217): CHtml::activeInputField('text', Object(User), 'email', Array)
#4 D:\yii\framework\web\widgets\CActiveForm.php(655): CHtml::activeTextField(Object(User), 'email', Array)
#5 D:\yii\webroot\protected\views\user\_form.php(32): CActiveForm->textField(Object(User), 'email', Array)
#6 D:\yii\framework\web\CBaseController.php(127): require('D:\yii\webroot\...')
#7 D:\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\yii\webroot\...', Array, true)
#8 D:\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\yii\webroot\...', Array, true)
#9 D:\yii\webroot\protected\views\user\create.php(18): CController->renderPartial('_form', Array)
#10 D:\yii\framework\web\CBaseController.php(127): require('D:\yii\webroot\...')
#11 D:\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\yii\webroot\...', Array, true)
#12 D:\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\yii\webroot\...', Array, true)
#13 D:\yii\framework\web\CController.php(783): CController->renderPartial('create', Array, true)
#14 D:\yii\webroot\protected\controllers\UserController.php(79): CController->render('create', Array)
#15 D:\yii\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#16 D:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 D:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 D:\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 D:\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#20 D:\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 D:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 D:\yii\framework\web\CController.php(292): CFilterChain->run()
#23 D:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 D:\yii\framework\web\CWebApplication.php(283): CController->run('create')
#25 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('user/create')
#26 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#27 D:\yii\webroot\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/index.php?r=user/create
HTTP_REFERER=http://127.0.0.1/index.php?r=user
---
2012/12/23 23:18:54 [error] [exception.CException] exception 'CException' with message 'Property "User.email" is not defined.' in D:\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('email')
#1 D:\yii\framework\web\helpers\CHtml.php(1809): CActiveRecord->__get('email')
#2 D:\yii\framework\zii\widgets\CDetailView.php(212): CHtml::value(Object(User), 'email')
#3 D:\yii\framework\web\CBaseController.php(174): CDetailView->run()
#4 D:\yii\webroot\protected\views\user\view.php(29): CBaseController->widget('zii.widgets.CDe...', Array)
#5 D:\yii\framework\web\CBaseController.php(127): require('D:\yii\webroot\...')
#6 D:\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\yii\webroot\...', Array, true)
#7 D:\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\yii\webroot\...', Array, true)
#8 D:\yii\framework\web\CController.php(783): CController->renderPartial('view', Array, true)
#9 D:\yii\webroot\protected\controllers\UserController.php(56): CController->render('view', Array)
#10 [internal function]: UserController->actionView('1')
#11 D:\yii\framework\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#12 D:\yii\framework\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#13 D:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#14 D:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#15 D:\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#16 D:\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#17 D:\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#18 D:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#19 D:\yii\framework\web\CController.php(292): CFilterChain->run()
#20 D:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#21 D:\yii\framework\web\CWebApplication.php(283): CController->run('view')
#22 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('user/view')
#23 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#24 D:\yii\webroot\index.php(13): CApplication->run()
#25 {main}
REQUEST_URI=/index.php?r=user/view&id=1
HTTP_REFERER=http://127.0.0.1/index.php?r=user/create
---
2012/12/24 00:09:05 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\yii\framework\web\auth\CAccessControlFilter.php:170
Stack trace:
#0 D:\yii\framework\web\auth\CAccessControlFilter.php(133): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 D:\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 D:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 D:\yii\framework\web\CController.php(292): CFilterChain->run()
#6 D:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\yii\framework\web\CWebApplication.php(283): CController->run('admin')
#8 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('user/admin')
#9 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\yii\webroot\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/admin
HTTP_REFERER=http://127.0.0.1/index.php?r=site/login
---
2012/12/24 00:10:27 [error] [exception.CException] exception 'CException' with message 'Property "User.email" is not defined.' in D:\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('email')
#1 D:\yii\framework\web\helpers\CHtml.php(2158): CActiveRecord->__get('email')
#2 D:\yii\framework\web\helpers\CHtml.php(1887): CHtml::resolveValue(Object(User), 'email')
#3 D:\yii\framework\web\helpers\CHtml.php(1217): CHtml::activeInputField('text', Object(User), 'email', Array)
#4 D:\yii\framework\web\widgets\CActiveForm.php(655): CHtml::activeTextField(Object(User), 'email', Array)
#5 D:\yii\webroot\protected\views\user\_search.php(26): CActiveForm->textField(Object(User), 'email', Array)
#6 D:\yii\framework\web\CBaseController.php(127): require('D:\yii\webroot\...')
#7 D:\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\yii\webroot\...', Array, true)
#8 D:\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\yii\webroot\...', Array, true)
#9 D:\yii\webroot\protected\views\user\admin.php(40): CController->renderPartial('_search', Array)
#10 D:\yii\framework\web\CBaseController.php(127): require('D:\yii\webroot\...')
#11 D:\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\yii\webroot\...', Array, true)
#12 D:\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\yii\webroot\...', Array, true)
#13 D:\yii\framework\web\CController.php(783): CController->renderPartial('admin', Array, true)
#14 D:\yii\webroot\protected\controllers\UserController.php(143): CController->render('admin', Array)
#15 D:\yii\framework\web\actions\CInlineAction.php(50): UserController->actionAdmin()
#16 D:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 D:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 D:\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 D:\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#20 D:\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 D:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 D:\yii\framework\web\CController.php(292): CFilterChain->run()
#23 D:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 D:\yii\framework\web\CWebApplication.php(283): CController->run('admin')
#25 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('user/admin')
#26 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#27 D:\yii\webroot\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/index.php?r=user/admin
HTTP_REFERER=http://127.0.0.1/index.php?r=site/login
---
2012/12/24 00:10:49 [error] [exception.CException] exception 'CException' with message 'Property "User.email" is not defined.' in D:\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('email')
#1 D:\yii\framework\web\helpers\CHtml.php(2158): CActiveRecord->__get('email')
#2 D:\yii\framework\web\helpers\CHtml.php(1887): CHtml::resolveValue(Object(User), 'email')
#3 D:\yii\framework\web\helpers\CHtml.php(1217): CHtml::activeInputField('text', Object(User), 'email', Array)
#4 D:\yii\framework\web\widgets\CActiveForm.php(655): CHtml::activeTextField(Object(User), 'email', Array)
#5 D:\yii\webroot\protected\views\user\_search.php(26): CActiveForm->textField(Object(User), 'email', Array)
#6 D:\yii\framework\web\CBaseController.php(127): require('D:\yii\webroot\...')
#7 D:\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\yii\webroot\...', Array, true)
#8 D:\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\yii\webroot\...', Array, true)
#9 D:\yii\webroot\protected\views\user\admin.php(40): CController->renderPartial('_search', Array)
#10 D:\yii\framework\web\CBaseController.php(127): require('D:\yii\webroot\...')
#11 D:\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\yii\webroot\...', Array, true)
#12 D:\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\yii\webroot\...', Array, true)
#13 D:\yii\framework\web\CController.php(783): CController->renderPartial('admin', Array, true)
#14 D:\yii\webroot\protected\controllers\UserController.php(143): CController->render('admin', Array)
#15 D:\yii\framework\web\actions\CInlineAction.php(50): UserController->actionAdmin()
#16 D:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 D:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 D:\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 D:\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#20 D:\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 D:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 D:\yii\framework\web\CController.php(292): CFilterChain->run()
#23 D:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 D:\yii\framework\web\CWebApplication.php(283): CController->run('admin')
#25 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('user/admin')
#26 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#27 D:\yii\webroot\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/index.php?r=user/admin
HTTP_REFERER=http://127.0.0.1/index.php?r=site/login
---
2012/12/24 00:11:05 [error] [exception.CException] exception 'CException' with message 'Property "User.email" is not defined.' in D:\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('email')
#1 D:\yii\framework\web\helpers\CHtml.php(2158): CActiveRecord->__get('email')
#2 D:\yii\framework\web\helpers\CHtml.php(1887): CHtml::resolveValue(Object(User), 'email')
#3 D:\yii\framework\web\helpers\CHtml.php(1217): CHtml::activeInputField('text', Object(User), 'email', Array)
#4 D:\yii\framework\web\widgets\CActiveForm.php(655): CHtml::activeTextField(Object(User), 'email', Array)
#5 D:\yii\webroot\protected\views\user\_search.php(26): CActiveForm->textField(Object(User), 'email', Array)
#6 D:\yii\framework\web\CBaseController.php(127): require('D:\yii\webroot\...')
#7 D:\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\yii\webroot\...', Array, true)
#8 D:\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\yii\webroot\...', Array, true)
#9 D:\yii\webroot\protected\views\user\admin.php(40): CController->renderPartial('_search', Array)
#10 D:\yii\framework\web\CBaseController.php(127): require('D:\yii\webroot\...')
#11 D:\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\yii\webroot\...', Array, true)
#12 D:\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\yii\webroot\...', Array, true)
#13 D:\yii\framework\web\CController.php(783): CController->renderPartial('admin', Array, true)
#14 D:\yii\webroot\protected\controllers\UserController.php(143): CController->render('admin', Array)
#15 D:\yii\framework\web\actions\CInlineAction.php(50): UserController->actionAdmin()
#16 D:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 D:\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 D:\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 D:\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#20 D:\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 D:\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 D:\yii\framework\web\CController.php(292): CFilterChain->run()
#23 D:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 D:\yii\framework\web\CWebApplication.php(283): CController->run('admin')
#25 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('user/admin')
#26 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#27 D:\yii\webroot\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/index.php?r=user/admin
HTTP_REFERER=http://127.0.0.1/index.php?r=user/
---
2012/12/31 18:25:25 [error] [php] Declaration of CMongodbHttpSession::regenerateID() should be compatible with that of CHttpSession::regenerateID() (D:\yii\webroot\protected\components\CMongodbHttpSession.php:4)
Stack trace:
#0 D:\yii\framework\YiiBase.php(198): import()
#1 D:\yii\framework\base\CModule.php(387): createComponent()
#2 D:\yii\framework\web\CWebApplication.php(202): CWebApplication->getComponent()
#3 D:\yii\framework\web\auth\CWebUser.php(194): CWebApplication->getSession()
#4 D:\yii\framework\base\CModule.php(388): CWebUser->init()
#5 D:\yii\framework\base\CModule.php(104): CWebApplication->getComponent()
#6 D:\yii\webroot\protected\controllers\SiteController.php(32): CWebApplication->__get()
#7 D:\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#8 D:\yii\framework\web\CController.php(309): CInlineAction->runWithParams()
#9 D:\yii\framework\web\CController.php(287): SiteController->runAction()
#10 D:\yii\framework\web\CController.php(266): SiteController->runActionWithFilters()
#11 D:\yii\framework\web\CWebApplication.php(283): SiteController->run()
#12 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController()
#13 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 D:\yii\webroot\index.php(13): CWebApplication->run()
REQUEST_URI=/
in D:\yii\webroot\protected\components\CMongodbHttpSession.php (4)
in D:\yii\webroot\protected\controllers\SiteController.php (32)
in D:\yii\webroot\index.php (13)
2012/12/31 20:03:47 [error] [exception.ActiveMongo_Exception] exception 'ActiveMongo_Exception' with message 'There is no information about the default DB name' in D:\yii\webroot\protected\vendors\mongodb\ActiveMongo.php:319
Stack trace:
#0 D:\yii\webroot\protected\vendors\mongodb\ActiveMongo.php(423): ActiveMongo->getDatabaseName()
#1 D:\yii\webroot\protected\vendors\mongodb\ActiveMongo.php(455): ActiveMongo->_getConnection()
#2 D:\yii\webroot\protected\vendors\mongodb\ActiveMongo.php(2081): ActiveMongo->_getCollection()
#3 D:\yii\webroot\protected\vendors\mongodb\ActiveMongo.php(1497): ActiveMongo->doQuery()
#4 D:\yii\webroot\protected\vendors\mongodb\ActiveMongo.php(1302): ActiveMongo->valid()
#5 D:\yii\webroot\protected\vendors\MongodbSession.php(48): ActiveMongo->count()
#6 D:\yii\webroot\protected\components\CMongodbHttpSession.php(22): MongodbSession::RegenerateID(true, 'j48s73ajari3stv...', '976p24eku3ua53l...', 1440)
#7 D:\yii\framework\web\auth\CWebUser.php(700): CMongodbHttpSession->regenerateID(true)
#8 D:\yii\framework\web\auth\CWebUser.php(228): CWebUser->changeIdentity('xiaofuqian2', 'xiaofuqian2', Array)
#9 D:\yii\webroot\protected\models\LoginForm.php(71): CWebUser->login(Object(UserIdentity), 0)
#10 D:\yii\webroot\protected\controllers\SiteController.php(94): LoginForm->login()
#11 D:\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#12 D:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#13 D:\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#14 D:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 D:\yii\framework\web\CWebApplication.php(283): CController->run('login')
#16 D:\yii\framework\web\CWebApplication.php(142): CWebApplication->runController('site/login')
#17 D:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#18 D:\yii\webroot\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://127.0.0.1/index.php?r=site/login
---
2013/01/12 11:09:03 [error] [exception.CException] exception 'CException' with message 'Alias "system.gii.GiiModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in D:\mu_project\code\framework\YiiBase.php:318
Stack trace:
#0 D:\mu_project\code\framework\YiiBase.php(198): YiiBase::import('system.gii.GiiM...', true)
#1 D:\mu_project\code\framework\base\CModule.php(283): YiiBase::createComponent('system.gii.GiiM...', 'gii', NULL, Array)
#2 D:\mu_project\code\framework\web\CWebApplication.php(338): CModule->getModule('gii')
#3 D:\mu_project\code\framework\web\CWebApplication.php(277): CWebApplication->createController('gii/')
#4 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController('gii/')
#5 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 D:\mu_project\code\webroot\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/index.php?r=gii/
---
2013/01/12 15:42:35 [error] [php] include(AdminWebUser.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (D:\mu_project\code\framework\YiiBase.php:423)
Stack trace:
#0 D:\mu_project\code\framework\YiiBase.php(198): import()
#1 D:\mu_project\code\framework\base\CModule.php(283): createComponent()
#2 D:\mu_project\code\framework\web\CWebApplication.php(338): CWebApplication->getModule()
#3 D:\mu_project\code\framework\web\CWebApplication.php(277): CWebApplication->createController()
#4 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController()
#5 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 D:\mu_project\code\webroot\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=admin
in D:\mu_project\code\webroot\index.php (13)
2013/01/12 15:43:48 [error] [php] include(AdminWebUser.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (D:\mu_project\code\framework\YiiBase.php:423)
Stack trace:
#0 D:\mu_project\code\framework\YiiBase.php(198): import()
#1 D:\mu_project\code\framework\base\CModule.php(283): createComponent()
#2 D:\mu_project\code\framework\web\CWebApplication.php(338): CWebApplication->getModule()
#3 D:\mu_project\code\framework\web\CWebApplication.php(277): CWebApplication->createController()
#4 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController()
#5 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 D:\mu_project\code\webroot\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=admin
in D:\mu_project\code\webroot\index.php (13)
2013/01/12 15:50:46 [error] [php] include(AdminWebUser.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (D:\mu_project\code\framework\YiiBase.php:423)
Stack trace:
#0 D:\mu_project\code\framework\YiiBase.php(198): import()
#1 D:\mu_project\code\framework\base\CModule.php(283): createComponent()
#2 D:\mu_project\code\framework\web\CWebApplication.php(338): CWebApplication->getModule()
#3 D:\mu_project\code\framework\web\CWebApplication.php(277): CWebApplication->createController()
#4 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController()
#5 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 D:\mu_project\code\webroot\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=admin
in D:\mu_project\code\webroot\index.php (13)
2013/01/12 15:53:12 [error] [php] Declaration of WebUser::login() should be compatible with that of CWebUser::login() (D:\mu_project\code\webroot\protected\components\WebUser.php:3)
Stack trace:
#0 D:\mu_project\code\framework\YiiBase.php(198): import()
#1 D:\mu_project\code\framework\base\CModule.php(387): createComponent()
#2 D:\mu_project\code\framework\base\CModule.php(104): CWebApplication->getComponent()
#3 D:\mu_project\code\webroot\protected\modules\admin\AdminModule.php(41): CWebApplication->__get()
#4 D:\mu_project\code\framework\web\CController.php(264): AdminModule->beforeControllerAction()
#5 D:\mu_project\code\framework\web\CWebApplication.php(283): DefaultController->run()
#6 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController()
#7 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#8 D:\mu_project\code\webroot\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=admin
in D:\mu_project\code\webroot\protected\components\WebUser.php (3)
in D:\mu_project\code\webroot\protected\modules\admin\AdminModule.php (41)
in D:\mu_project\code\webroot\index.php (13)
2013/01/13 17:20:41 [error] [php] Declaration of AdminWebUser::login() should be compatible with that of CWebUser::login() (D:\mu_project\code\webroot\protected\modules\admin\components\AdminWebUser.php:3)
Stack trace:
#0 D:\mu_project\code\framework\YiiBase.php(198): import()
#1 D:\mu_project\code\framework\base\CModule.php(387): createComponent()
#2 D:\mu_project\code\framework\base\CModule.php(104): CWebApplication->getComponent()
#3 D:\mu_project\code\webroot\protected\modules\admin\AdminModule.php(41): CWebApplication->__get()
#4 D:\mu_project\code\framework\web\CController.php(264): AdminModule->beforeControllerAction()
#5 D:\mu_project\code\framework\web\CWebApplication.php(284): SiteController->run()
#6 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController()
#7 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#8 D:\mu_project\code\webroot\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=admin/site
in D:\mu_project\code\webroot\protected\modules\admin\components\AdminWebUser.php (3)
in D:\mu_project\code\webroot\protected\modules\admin\AdminModule.php (41)
in D:\mu_project\code\webroot\index.php (13)
2013/01/13 17:21:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in D:\mu_project\code\framework\web\CController.php:484
Stack trace:
#0 D:\mu_project\code\framework\web\CController.php(271): CController->missingAction('login')
#1 D:\mu_project\code\framework\web\CWebApplication.php(284): CController->run('login')
#2 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController('admin/default/l...')
#3 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 D:\mu_project\code\webroot\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=admin/default/login
---
2013/01/13 17:26:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in D:\mu_project\code\framework\web\CController.php:484
Stack trace:
#0 D:\mu_project\code\framework\web\CController.php(271): CController->missingAction('login')
#1 D:\mu_project\code\framework\web\CWebApplication.php(284): CController->run('login')
#2 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController('admin/default/l...')
#3 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 D:\mu_project\code\webroot\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=admin/default/login
---
2013/01/13 19:36:45 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'mydb.tbl_user' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_user`.
in D:\mu_project\code\webroot\protected\modules\admin\models\User.php (21)
in D:\mu_project\code\webroot\protected\modules\admin\components\UserIdentity.php (28)
in D:\mu_project\code\webroot\protected\models\LoginForm.php (52)
2013/01/13 19:36:45 [error] [exception.CDbException] exception 'CDbException' with message 'The table "tbl_user" for active record class "User" cannot be found in the database.' in D:\mu_project\code\framework\db\ar\CActiveRecord.php:2266
Stack trace:
#0 D:\mu_project\code\framework\db\ar\CActiveRecord.php(383): CActiveRecordMetaData->__construct(Object(User))
#1 D:\mu_project\code\webroot\protected\modules\admin\models\User.php(21): CActiveRecord::model('User')
#2 D:\mu_project\code\webroot\protected\modules\admin\components\UserIdentity.php(28): User::model()
#3 D:\mu_project\code\webroot\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 D:\mu_project\code\framework\validators\CInlineValidator.php(43): LoginForm->authenticate('password', Array)
#5 D:\mu_project\code\framework\validators\CValidator.php(214): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 D:\mu_project\code\framework\base\CModel.php(158): CValidator->validate(Object(LoginForm), NULL)
#7 D:\mu_project\code\webroot\protected\modules\admin\controllers\SiteController.php(94): CModel->validate()
#8 D:\mu_project\code\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#9 D:\mu_project\code\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 D:\mu_project\code\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 D:\mu_project\code\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\mu_project\code\framework\web\CWebApplication.php(284): CController->run('login')
#13 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController('admin/site/logi...')
#14 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 D:\mu_project\code\webroot\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=admin/site/login
HTTP_REFERER=http://127.0.0.1/index.php?r=admin/site/login
---
2013/01/13 19:37:15 [error] [php] include(IASolrConnection.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (D:\mu_project\code\framework\YiiBase.php:423)
Stack trace:
#0 D:\mu_project\code\webroot\protected\packages\solr\ASolrConnection.php(8): spl_autoload_call()
#1 D:\mu_project\code\framework\YiiBase.php(316): require()
#2 D:\mu_project\code\framework\YiiBase.php(198): import()
#3 D:\mu_project\code\framework\base\CModule.php(387): createComponent()
#4 D:\mu_project\code\framework\gii\generators\model\ModelCode.php(428): CWebApplication->getComponent()
#5 D:\mu_project\code\framework\gii\generators\model\ModelCode.php(407): ModelCode->isDbConnection()
#6 D:\mu_project\code\framework\gii\generators\model\views\index.php(43): ModelCode->getConnectionList()
#7 D:\mu_project\code\framework\web\CBaseController.php(127): require()
#8 D:\mu_project\code\framework\web\CBaseController.php(96): ModelGenerator->renderInternal()
#9 D:\mu_project\code\framework\web\CController.php(870): ModelGenerator->renderFile()
#10 D:\mu_project\code\framework\web\CController.php(783): ModelGenerator->renderPartial()
#11 D:\mu_project\code\framework\gii\CCodeGenerator.php(73): ModelGenerator->render()
#12 D:\mu_project\code\framework\web\actions\CInlineAction.php(50): ModelGenerator->actionIndex()
#13 D:\mu_project\code\framework\web\CController.php(309): CInlineAction->runWithParams()
#14 D:\mu_project\code\framework\web\CController.php(287): ModelGenerator->runAction()
#15 D:\mu_project\code\framework\web\CController.php(266): ModelGenerator->runActionWithFilters()
#16 D:\mu_project\code\framework\web\CWebApplication.php(284): ModelGenerator->run()
#17 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController()
#18 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#19 D:\mu_project\code\webroot\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=gii/model
in D:\mu_project\code\webroot\protected\packages\solr\ASolrConnection.php (8)
in D:\mu_project\code\webroot\index.php (13)
2013/01/13 19:41:39 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'mydb.tbl_user' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_user`.
in D:\mu_project\code\webroot\protected\modules\admin\models\User.php (21)
in D:\mu_project\code\webroot\protected\modules\admin\components\UserIdentity.php (28)
in D:\mu_project\code\webroot\protected\models\LoginForm.php (52)
2013/01/13 19:41:39 [error] [exception.CDbException] exception 'CDbException' with message 'The table "tbl_user" for active record class "User" cannot be found in the database.' in D:\mu_project\code\framework\db\ar\CActiveRecord.php:2266
Stack trace:
#0 D:\mu_project\code\framework\db\ar\CActiveRecord.php(383): CActiveRecordMetaData->__construct(Object(User))
#1 D:\mu_project\code\webroot\protected\modules\admin\models\User.php(21): CActiveRecord::model('User')
#2 D:\mu_project\code\webroot\protected\modules\admin\components\UserIdentity.php(28): User::model()
#3 D:\mu_project\code\webroot\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 D:\mu_project\code\framework\validators\CInlineValidator.php(43): LoginForm->authenticate('password', Array)
#5 D:\mu_project\code\framework\validators\CValidator.php(214): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 D:\mu_project\code\framework\base\CModel.php(158): CValidator->validate(Object(LoginForm), NULL)
#7 D:\mu_project\code\webroot\protected\modules\admin\controllers\SiteController.php(94): CModel->validate()
#8 D:\mu_project\code\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#9 D:\mu_project\code\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 D:\mu_project\code\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 D:\mu_project\code\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\mu_project\code\framework\web\CWebApplication.php(284): CController->run('login')
#13 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController('admin/site/logi...')
#14 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 D:\mu_project\code\webroot\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=admin/site/login
HTTP_REFERER=http://127.0.0.1/index.php?r=admin/site/login
---
2013/01/13 19:43:03 [error] [exception.CException] exception 'CException' with message 'Property "User.password" is not defined.' in D:\mu_project\code\framework\base\CComponent.php:131
Stack trace:
#0 D:\mu_project\code\framework\db\ar\CActiveRecord.php(144): CComponent->__get('password')
#1 D:\mu_project\code\webroot\protected\modules\admin\components\UserIdentity.php(31): CActiveRecord->__get('password')
#2 D:\mu_project\code\webroot\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 D:\mu_project\code\framework\validators\CInlineValidator.php(43): LoginForm->authenticate('password', Array)
#4 D:\mu_project\code\framework\validators\CValidator.php(214): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 D:\mu_project\code\framework\base\CModel.php(158): CValidator->validate(Object(LoginForm), NULL)
#6 D:\mu_project\code\webroot\protected\modules\admin\controllers\SiteController.php(94): CModel->validate()
#7 D:\mu_project\code\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#8 D:\mu_project\code\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#9 D:\mu_project\code\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#10 D:\mu_project\code\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\mu_project\code\framework\web\CWebApplication.php(284): CController->run('login')
#12 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController('admin/site/logi...')
#13 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 D:\mu_project\code\webroot\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/index.php?r=admin/site/login
HTTP_REFERER=http://127.0.0.1/index.php?r=admin/site/login
---
2013/01/14 21:21:47 [error] [php] include(WebUser.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (D:\mu_project\code\framework\YiiBase.php:423)
Stack trace:
#0 D:\mu_project\code\framework\YiiBase.php(198): import()
#1 D:\mu_project\code\framework\base\CModule.php(387): createComponent()
#2 D:\mu_project\code\framework\base\CModule.php(104): CWebApplication->getComponent()
#3 D:\mu_project\code\webroot\protected\views\layouts\main.php(35): CWebApplication->__get()
#4 D:\mu_project\code\framework\web\CBaseController.php(127): require()
#5 D:\mu_project\code\framework\web\CBaseController.php(96): SiteController->renderInternal()
#6 D:\mu_project\code\framework\web\widgets\CContentDecorator.php(77): SiteController->renderFile()
#7 D:\mu_project\code\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate()
#8 D:\mu_project\code\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput()
#9 D:\mu_project\code\framework\web\CBaseController.php(207): CContentDecorator->run()
#10 D:\mu_project\code\framework\web\CBaseController.php(301): SiteController->endWidget()
#11 D:\mu_project\code\webroot\protected\views\layouts\column1.php(6): SiteController->endContent()
#12 D:\mu_project\code\framework\web\CBaseController.php(127): require()
#13 D:\mu_project\code\framework\web\CBaseController.php(96): SiteController->renderInternal()
#14 D:\mu_project\code\framework\web\CController.php(785): SiteController->renderFile()
#15 D:\mu_project\code\webroot\protected\modules\admin\controllers\SiteController.php(98): SiteController->render()
#16 D:\mu_project\code\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#17 D:\mu_project\code\framework\web\CController.php(309): CInlineAction->runWithParams()
#18 D:\mu_project\code\framework\web\CController.php(287): SiteController->runAction()
#19 D:\mu_project\code\framework\web\CController.php(266): SiteController->runActionWithFilters()
#20 D:\mu_project\code\framework\web\CWebApplication.php(284): SiteController->run()
#21 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController()
#22 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#23 D:\mu_project\code\webroot\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=admin/site/login
in D:\mu_project\code\webroot\protected\views\layouts\main.php (35)
in D:\mu_project\code\webroot\protected\views\layouts\column1.php (6)
in D:\mu_project\code\webroot\protected\modules\admin\controllers\SiteController.php (98)
2013/01/14 21:31:07 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.user_id" is not defined.' in D:\mu_project\code\framework\base\CComponent.php:131
Stack trace:
#0 D:\mu_project\code\webroot\protected\modules\admin\components\UserIdentity.php(62): CComponent->__get('user_id')
#1 D:\mu_project\code\framework\web\auth\CWebUser.php(224): UserIdentity->getId()
#2 D:\mu_project\code\webroot\protected\modules\admin\components\AdminWebUser.php(19): CWebUser->login(Object(UserIdentity), 0)
#3 D:\mu_project\code\webroot\protected\modules\admin\models\LoginForm.php(71): AdminWebUser->login(Object(UserIdentity), 0)
#4 D:\mu_project\code\webroot\protected\modules\admin\controllers\SiteController.php(94): LoginForm->login()
#5 D:\mu_project\code\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#6 D:\mu_project\code\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#7 D:\mu_project\code\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#8 D:\mu_project\code\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 D:\mu_project\code\framework\web\CWebApplication.php(284): CController->run('login')
#10 D:\mu_project\code\framework\web\CWebApplication.php(142): CWebApplication->runController('admin/site/logi...')
#11 D:\mu_project\code\framework\base\CApplication.php(162): CWebApplication->processRequest()
#12 D:\mu_project\code\webroot\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=admin/site/login
HTTP_REFERER=http://127.0.0.1/index.php?r=admin/site/login
---
